src/test/java/playground/AppTest.java

```diff
@@ -1,15 +1,65 @@
-package playground;
+  package playground;
 
-import static org.junit.Assert.assertNotNull;
+class Shapes
+{
+  public static void main(String arg[])
+  {
+    Rectangle r = new Rectangle(10, 4);
```

Comment:
How can we use polymorphism here? Will that simplify it?

Reply:
yes i can will update the code again in sometime